docs/rules/aws_iam_group_policy_too_long.md

```diff
@@ -0,0 +1,34 @@
+# aws_iam_group_policy_too_long
+
+This makes sure that an IAM group policy is not longer than the 5120 AWS character limit.
+## Example
+
+```hcl
+resource "aws_iam_group_policy" "policy" {
+	name        = "test_policy"
+	group       = "test_group"
+	description = "My test policy"
+	policy = <<EOF
+	{
+		STRING LONGER THAN 5120
+	}
+EOF
+}
+```
+
+```
+$ tflint
+1 issue(s) found:
+Error: The policy length is %d characters and is limited to 5120 characters. (aws_iam_policy_too_long_policy)
+  on template.tf line 6:
+   6:   policy = "{POLICY}" // Policy is too long,!
+
+```
+
+## Why
+
+Terraform does not check against this rule, but it will error if an apply is attempted.
```

Comment:
It would be nicer to have a link to the underlying documentation:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html#reference_iam-quotas-entity-length